include/api/CAnomalyJobConfig.h

```diff
@@ -243,6 +246,8 @@ class API_EXPORT CAnomalyJobConfig {
 
         core_t::TTime bucketSpan() const { return m_BucketSpan; }
 
+        core_t::TTime modelPruneWindow() const { return m_ModelPruneWindow; }
```

Comment:
It will be quite confusing to future maintainers that sometimes "prune window" is expressed as a number of buckets and other times it's expressed as a period of time.  It would help those future maintainers if every related method and member variable had a Doxygen comment saying that sometimes "prune window" is expressed as a number of buckets and other times it's expressed as a period of time, and which of the two cases applies for the method/variable that the comment relates to.  (I can see that some already do, but not this one.)